server/src/index.ts

```diff
@@ -219,6 +246,8 @@ const apolloServer = new ApolloServer({
 });
 apolloServer.applyMiddleware({ app });
 
+apolloServer.installSubscriptionHandlers(http);
```

Comment:
Do you think we should remove socket.io in this pr or in another one?

Reply:
@pradel I am for doing this in a separate PR. Created and issue for this, so we don't forget : https://github.com/ledokku/ledokku/issues/133 .